lib/active_triples/rdf_source.rb

```diff
@@ -102,23 +88,35 @@ def writable?
     # @todo move this logic out to a Builder?
     def initialize(*args, &block)
       resource_uri = args.shift unless args.first.is_a?(Hash)
-      self.parent = args.shift unless args.first.is_a?(Hash)
+      unless args.first.is_a?(Hash) || args.empty?
+        set_persistence_strategy(ParentStrategy)
```

Comment:
I think this is a problem with set_persistence_strategy accepting a class rather than an instance of the strategy. ParentStrategy is -totally- broken without a defined parent, so I'd expect to see it in the constructor. The alternative is something like

```ruby
class ParentStrategyFactory
  attr_reader :parent
  def initialize(parent)
    @parent = parent
  end

  def new(obj)
    ParentStrategy.new(obj, parent)
  end
end
```
If you want to maintain the interface.